website/docs/cloud-docs/run/ui.mdx

```diff
@@ -25,9 +25,9 @@ The Terraform code for a normal run always comes from version control, and is al
 
 ## Automatically Starting Runs
 
-In a workspace linked to a VCS repo, runs start automatically when you merge or commit changes to version control.
+In a workspace linked to a VCS repository, runs start automatically when you merge or commit changes to version control.
 
-A workspace is linked to one branch of its repository, and ignores changes to other branches. Workspaces can also ignore some changes within their branch: if a Terraform working directory is configured, Terraform Cloud assumes that only some of the content in the repository is relevant to Terraform, and ignores changes outside of that content. (This behavior can be configured; for details, see [Settings: Automatic Run Triggering](/cloud-docs/workspaces/settings/vcs#automatic-run-triggering).)
+A workspace is linked to one branch of a VCS repository and ignores changes to other branches. Workspaces can also ignore some changes within their branch. If a Terraform working directory is configured, Terraform Cloud assumes that only some of the content in the repository is relevant to Terraform, and ignores changes outside of that content. You can also specify which files and directories within your repository trigger runs. Refer to [Automatic Run Triggering](/cloud-docs/workspaces/settings/vcs#automatic-run-triggering) for details.
```

Comment:
@matejrisek ah, I didn't realize there was already a link to the Run Triggers section that we're working on. This is perfect, I just modified it a bit so it adheres to our style guide.

Reply:
```
If a Terraform working directory is configured, Terraform Cloud assumes that only some of the content in the repository is relevant to Terraform
```

I've missed this part here - with the introduction of `trigger-patterns` we won't insist on the dependency between working directory and triggers. Trigger prefixes on the other hand will still take working directory into consideration but we won't insist on specifying a working directory as a prerequisite to input `trigger-prefixes`.

We would probably want to remove/reword the above statement from this part of documentation.

Reply:
got it! I'm going to push up a change to remove!